modules/ROOT/pages/clauses/match.adoc

```diff
@@ -262,151 +228,262 @@ RETURN a, b
 |===
-| type(r)
-| "OLD FRIENDS"
-|Rows: 1
+| path
+| (:Person {name: "Charlie Sheen"})-[:ACTED_IN {role: "Bud Fox"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+| (:Person {name: "Martin Sheen"})-[:ACTED_IN {role: "Carl Fox"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+| (:Person {name: "Michael Douglas"})-[:ACTED_IN {role: "Gordon Gekko"}]->(:Movie {title: "Wall Street"})<-[:DIRECTED]-(:Person {name: "Oliver Stone"})
+|Rows: 3
 |===
 
+For more information about how `MATCH` is used to find patterns of varying complexity (including xref:patterns/variable-length-patterns.adoc#quantified-path-patterns[quantified path patterns], xref:patterns/variable-length-patterns.adoc#quantified-relationships[quantified relationships], and the xref:patterns/shortest-paths.adoc[shortest paths] between nodes), see the section on xref::patterns/index.adoc[Patterns].
 
-[[multiple-rels]]
-=== Multiple relationships
+== Multiple MATCH clauses, the WITH clause, and clause composition
 
-Relationships can be expressed by using multiple statements in the form of `()--()`, or they can be strung together.
-For example:
+In Cypher, a query’s behavior is defined by its clauses.
+Each clause takes the current graph state and a table of intermediate results, processes them, and passes the updated graph state and results to the next clause.
+The first clause starts with the graph's initial state and an empty table, while the final clause produces the query's result.
 
-.Query
-[source, cypher, indent=0]
+.Chaining consecutive `MATCH` clauses
+[source, cypher]
 ----
-MATCH (charlie {name: 'Charlie Sheen'})-[:ACTED_IN]->(movie)<-[:DIRECTED]-(director)
-RETURN movie.title, director.name
+MATCH (:Person {name: 'Martin Sheen'})-[:ACTED_IN]->(movie:Movie) // <1>
+MATCH (director:Person)-[:DIRECTED]->(movie) // <2>
+RETURN director.name AS director, movie.title AS movieTitle
 ----
-
-Returns the movie in which `Charlie Sheen` acted and its director.
+<1> The result of the first `MATCH` clause is the variable `movie` which holds all the `Movies` that `Martin Sheen` has `ACTED_IN`.
+<2> The second `MATCH` clause uses the `movie` variable to find any `Person` node with a `DIRECTED` relationship to those `Movie` nodes that `Martin Sheen` has `ACTED_IN`.
 
 .Result
 [role="queryresult",options="header,footer",cols="2*<m"]
 |===
-| movie.title | director.name
-| "Wall Street" | "Oliver Stone"
-2+|Rows: 1
+| director | movieTitle
+
+| "Oliver Stone" | "Wall Street"
+| "Rob Reiner"   | "The American President"
+
+2+d| Rows: 2
 |===
+
+A variable can be implicitly carried over to the following clause by being referenced in another operation.
+A variable can also be explicitly passed to the following clause using the xref:clauses/with.adoc[`WITH`] clause.
+If a variable is neither implicitly nor explicitly carried over to its following clause, it will be discarded and unavailable for reference later in the query.
```

Comment:
```suggestion
If a variable is neither implicitly nor explicitly carried over to its following clause, it will be discarded and is not available for reference later in the query.
```